weather-card/weather-card-editor.js

```diff
@@ -108,6 +183,27 @@ export class WeatherCardEditor extends LitElement {
     }
     fireEvent(this, "config-changed", { config: this._config });
   }
+
+  _checkChanged(ev) {
```

Comment:
Let's combine this with `_valueChanged` as it is mostly the same code.

Reply:
Will do, have also combined the two methods.